WordPress/Classes/ViewRelated/NUX/SignupDomainSuggestionViewController.swift

```diff
@@ -0,0 +1,241 @@
+import UIKit
+
+class SignupDomainSuggestionViewController: UITableViewController {
+    var helpBadge: WPNUXHelpBadgeLabel!
+    var helpButton: UIButton!
+
+    open var siteName: String?
+
+    var service: DomainsService?
+    private var siteTitleSuggestions: [String] = []
+    private var searchSuggestions: [String] = []
+    private var isSearching: Bool = false
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        tableView.register(UINib(nibName: "SiteCreationDomainSearchTableViewCell", bundle: nil), forCellReuseIdentifier: SiteCreationDomainSearchTableViewCell.cellIdentifier)
+        tableView.register(UINib(nibName: "SiteCreationDomainsActivityTableViewCell", bundle: nil), forCellReuseIdentifier: SiteCreationDomainsActivityTableViewCell.cellIdentifier)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = WPStyleGuide.greyLighten30()
```

Comment:
Just wondering - would this be better as `WPStyleGuide.configureColors(for: view, andTableView: tableView)`?